lib/generators/notify_user/install/templates/create_notify_user_deliveries.rb

```diff
@@ -0,0 +1,15 @@
+class CreateNotifyUserDeliveries < ActiveRecord::Migration
+  def change
+    create_table :notify_user_deliveries do |t|
+      t.datetime :sent_at
+      t.string :channel
+      t.integer :deliver_in
+
+      t.integer :notification_id
+
+      t.timestamps
+    end
+
+    add_index :notify_user_deliveries, :notification_id
```

Comment:
Should this have an index on `:sent_at` too, as you're querying for unsent ones?